sdk/servicebus/azure-messaging-servicebus/migration-guide.md

```diff
@@ -395,11 +395,78 @@ ServiceBusSessionReceiverClient sessionClient = new ServiceBusClientBuilder()
 
 ServiceBusReceiverClient receiverClient = sessionClient.acceptSession("my-session-id");
 ```
-## Upcoming features
- - [Cross entity transactions](https://docs.microsoft.com/azure/service-bus-messaging/service-bus-transactions#transfers-and-send-via) 
- to support transaction across different entities.
- - [Different AMQP body section](https://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#type-amqp-sequence) 
- support for 'Amqp Sequence' and 'Amqp Value'. 
+### Working with transactions across entities
+This feature is used when your transaction scope spans across different Service Bus entities. It is achieved by routing
+all the messages through one 'send-via' entity on server side.
+
+Previously, the 'send-via' entity used to be specified as an argument in 'ClientFactory.createMessageSenderFromEntityPath()' call.
+
+```java
+    String intermediateQueue = "intermediate-queue"; // this is 'send-via' queue
+    final String destination1 = "destination-1";
+    final String destination2 = "destination-2";
+    IMessageSender intermediateSender = ClientFactory.createMessageSenderFromEntityPath(factory, intermediateQueue);
+    IMessageReceiver intermediateReceiver = ClientFactory.createMessageReceiverFromEntityPath(factory, intermediateQueue, ReceiveMode.PEEKLOCK);
+    IMessageSender destination1Sender = ClientFactory.createMessageSenderFromEntityPath(factory, destination1);
+    IMessageSender destination1ViaSender = ClientFactory.createTransferMessageSenderFromEntityPathAsync(factory, destination1, intermediateQueue).get();
+ 
+    String destination2 = TestUtils.randomizeEntityName(this.getEntityNamePrefix());
```

Comment:
`TestUtils.` seems intense for a migration guide. I would input some random name.